components/TextFields/src/MDCTextInputControllerDefault.m

```diff
@@ -909,7 +918,12 @@ - (void)textInputDidBeginEditing:(NSNotification *)note {
 }
 
 - (void)textInputDidChange:(NSNotification *)note {
-  [self updateLayout];
```

Comment:
Do we no longer need to call `[self updateLayout]` here? Was it overkill previously?